src/gui/sdlmain.cpp

```diff
@@ -2332,23 +2330,15 @@ int main(int argc, char* argv[]) {
 			Sleep(5000);
 #endif
 		}
-
-	}
-	catch (int){
-		; //nothing, pressed killswitch
 	}
-	catch(...){
-		; // Unknown error, let's just exit.
+	catch (...) {
+		// just exit
+		rcode = 1;
 	}
 #if defined (WIN32)
 	sticky_keys(true); //Might not be needed if the shutdown function switches to windowed mode, but it doesn't hurt
 #endif
-	// Release the mouse if it's still captured
-	if (mouse_is_captured) {
-		GFX_ToggleMouseCapture();
-	}
```

Comment:
Original code mentioned that in some cases the mouse might not be returned properly (the cursor is always uncaptured and made-visible on SDL tear-down). Also, this never happened during testing the updated mouse capture code on Linux, macOS, and Window 7 and 10 under SDL2; so suggest we remove this SDL 1.2.x band-aid and let new issues be flagged if needed.